ui/src/PartCommands.cpp

```
@@ -0,0 +1,696 @@
+
+  // ── GRID PATTERN ──────────────────────────────────────────────────────────
+  {
+    CommandDescriptor c = base("part.pattern_grid", "Grid Pattern", "PATTERN",
+                               SelectionSignature::exactly(EntityKind::Body, 1));
+    c.schema.push_back(ParamSpec{"nx", ParamType::Number, true, 2.0, ""});
+    c.schema.push_back(ParamSpec{"ny", ParamType::Number, true, 2.0, ""});
+    c.schema.push_back(ParamSpec{"dx", ParamType::Number, true, 10.0, ""});
+    c.schema.push_back(ParamSpec{"dy", ParamType::Number, true, 10.0, ""});
+    c.preview = PreviewPolicy::OnDemand;
+    c.enabled = [d](const CommandContext& ctx) {
+      const double nx = num(ctx, "nx", 0.0);
```

Clang (14.0.6)

[warning] variable 'nx' is not initialized

(cppcoreguidelines-init-variables)